internal/api/magic_link.go

```diff
@@ -128,7 +128,7 @@ func (a *API) MagicLink(w http.ResponseWriter, r *http.Request) error {
 		flowState           *models.FlowState
 		codeChallengeMethod models.CodeChallengeMethod
 	)
-	if isPKCEFlow(models.PKCEFlow) {
+	if isPKCEFlow(flowType) {
```

Comment:
this is a big one because it causes the all implicit flow magiclinks to fail with a 500